### ISSUE TITLE
object_detection, What is default_ranges_min and default_ranges_max when quantifying the model under object_detection？

### ISSUE THREAD

**Issue by yeyupiaoling** [tensorflow/models#8172]
## environment
 - Ubuntu 16.04
 - Python 3.7
 - Tensorflow-GPU 1.14.0

## issue
What is default_ranges_min and default_ranges_max when quantifying the model under object_detection？
```
bazel-bin/tensorflow/lite/toco/toco \
--input_file=/mnt/d/tmp/tflite_graph.pb \
--output_file=/mnt/d/tmp/ssd_mobilenet_v2.tflite \
--input_shapes=1,300,300,3 \
--input_arrays=normalized_input_image_tensor \
--output_arrays='TFLite_Detection_PostProcess','TFLite_Detection_PostProcess:1','TFLite_Detection_PostProcess:2','TFLite_Detection_PostProcess:3'  \
--inference_type=QUANTIZED_UINT8 \
--mean_values=128 \
--std_values=128 \
--default_ranges_min=? \
--default_ranges_max=? \
--change_concat_input_ranges=false \
--allow_custom_ops
```

**Comment by vyasmansh** [comment]
When specified, they will be used as default (min, max) range boundaries for all activation arrays that lack (min, max) range information, thus allowing for quantization to proceed.

See for more :- [TOCO Flags](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/toco/toco_flags.proto)

Also :-  
To get help with issues you may encounter using the Tensorflow Object Detection API, create a new question on StackOverflow with the tags "tensorflow" and "object-detection".

Please report bugs (actually broken code, **not usage questions**) to the tensorflow/models GitHub issue tracker, prefixing the issue name with "object_detection".

Please check FAQ for frequently asked questions before reporting an issue.

**Comment by vyasmansh** [comment]
Such questions will be resolved after merging of #8103

**Comment by vyasmm** [comment]
When specified, they will be used as default (min, max) range boundaries for all activation arrays that lack (min, max) range information, thus allowing for quantization to proceed.

See for more :- [TOCO Flags](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/toco/toco_flags.proto)

Also :-  
To get help with issues you may encounter using the Tensorflow Object Detection API, create a new question on StackOverflow with the tags "tensorflow" and "object-detection".

Please report bugs (actually broken code, **not usage questions**) to the tensorflow/models GitHub issue tracker, prefixing the issue name with "object_detection".

**Comment by vyasmm** [comment]
Such questions will be resolved after merging of #8103
